### PR TITLE
fix app.trunk.io links

### PR DESCRIPTION
**Pull request by joshmarinacci**
make app links to app.trunk.io be https
make all flaky test links have the correct flaky+tests intent 
make all code quality links have the correct code+quality intent

### CODE REVIEW

File: code-quality/ci-setup/get-started.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ description: Automate your code quality enforcement with just a few clicks.
...
+Trunk Code Quality has the ability to post its results to the [Trunk Code Quality web app](https://app.trunk.io/login?intent=code+quality). This will enable you to view your repository's Code Quality history over time so you can track the trend of issues in your code, as well as browse the issues in your repository to help you understand which issues should be prioritized to fix.
 
 ## Connect your Trunk organization to GitHub
 
-Sign up at [app.trunk.io](https://app.trunk.io), create a Trunk organization, and connect it to your repositories. You will need to grant the following [GitHub App permissions](../../administration/github-app-permissions.md).
+Sign up at [app.trunk.io](https://app.trunk.io/?intent=code+quality), create a Trunk organization, and connect it to your repositories. You will need to grant the following [GitHub App permissions](../../administration/github-app-permissions.md).
```
Comment (gewenyu99) [inline code-quality/ci-setup/get-started.md:11]: I don't think `https://app.trunk.io/?intent=code+quality` does anything

File: merge-queue/set-up-trunk-merge/readme.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ Minimal set up is required to get started with Trunk Merge Queue, as it syncs wi
 
 ### Connect your Trunk Organization to GitHub
 
-Sign up at [app.trunk.io](https://app.trunk.io/signup?intent=merge\&tr\_s=mergesetup\&tr\_l=1), create a Trunk organization, and connect it to your GitHub repositories. If your repository is already connected to your Trunk organization, you can skip this step.
+Sign up at [app.trunk.io](https://app.trunk.io/signup?intent=merge&tr_s=mergesetup&tr_l=1), create a Trunk organization, and connect it to your GitHub repositories. If your repository is already connected to your Trunk organization, you can skip this step.
```
Comment (gewenyu99) [inline merge-queue/set-up-trunk-merge/readme.md:11]: Is this correct? Have to checked with Phil?

Comment (gewenyu99) [reply merge-queue/set-up-trunk-merge/readme.md:11]: Okay I think this doesn't work.

File: code-quality/ci-setup/get-started.md (hunk 1 above)
Comment (joshmarinacci) [reply code-quality/ci-setup/get-started.md]: Yeah, weird.  I've changed them all to have the /login/

File: merge-queue/set-up-trunk-merge/readme.md (hunk 2 above)
Comment (joshmarinacci) [reply merge-queue/set-up-trunk-merge/readme.md:11]: I think they are old tracking links for marketing.

**Review by gewenyu99 — APPROVED** [review]

File: merge-queue/set-up-trunk-merge/readme.md (hunk 2 above)
Comment (gewenyu99) [inline merge-queue/set-up-trunk-merge/readme.md:11]:
```suggestion
Sign up at [app.trunk.io](https://app.trunk.io/login?intent=merge&tr_s=mergesetup&tr_l=1), create a Trunk organization, and connect it to your GitHub repositories. If your repository is already connected to your Trunk organization, you can skip this step.
```

**Review by gewenyu99 — APPROVED** [review]

**Pull request merged**